man/systemd.netdev.xml

```diff
@@ -571,6 +571,16 @@
           <xi:include href="version-info.xml" xpointer="v248"/>
         </listitem>
       </varlistentry>
+     <varlistentry>
+        <term><varname>BroadcastThresholdLimit=</varname></term>
```

Comment:
ThresholdLimit sounds redundant, as both Threshold and Limit mean similar.
How about `BroadcastQueueThreashold=` or so?

Reply:
I think it's better `BroadcastQueueThreasholdLimit=`